pkg/workload/schemachange/optype.go

```diff
@@ -53,53 +54,143 @@ func init() {
-	addConstraint                         // ALTER TABLE <table> ADD CONSTRAINT <constraint> <def>
-	addForeignKeyConstraint               // ALTER TABLE <table> ADD CONSTRAINT <constraint> FOREIGN KEY (<column>) REFERENCES <table> (<column>)
-	addRegion                             // ALTER DATABASE <db> ADD REGION <region>
-	addUniqueConstraint                   // ALTER TABLE <table> ADD CONSTRAINT <constraint> UNIQUE (<column>)
+	// Non-DDL operations
 
-	alterTableLocality // ALTER TABLE <table> LOCALITY <locality>
+	insertRow  opType = iota // INSERT INTO <table> (<cols>) VALUES (<values>)
+	selectStmt               // SELECT..
+	validate                 // validate all table descriptors
 
-	createIndex    // CREATE INDEX <index> ON <table> <def>
-	createSequence // CREATE SEQUENCE <sequence> <def>
-	createTable    // CREATE TABLE <table> <def>
-	createTableAs  // CREATE TABLE <table> AS <def>
-	createView     // CREATE VIEW <view> AS <def>
-	createEnum     // CREATE TYPE <type> ENUM AS <def>
-	createSchema   // CREATE SCHEMA <schema>
+	// DDL operations
+
+	// Rename operations all rolled up into a single tree element and can't
+	// easily be deduced by reflect so they're manually added here.
 
-	dropColumn        // ALTER TABLE <table> DROP COLUMN <column>
-	dropColumnDefault // ALTER TABLE <table> ALTER [COLUMN] <column> DROP DEFAULT
-	dropColumnNotNull // ALTER TABLE <table> ALTER [COLUMN] <column> DROP NOT NULL
-	dropColumnStored  // ALTER TABLE <table> ALTER [COLUMN] <column> DROP STORED
-	dropConstraint    // ALTER TABLE <table> DROP CONSTRAINT <constraint>
-	dropIndex         // DROP INDEX <index>@<table>
-	dropSequence      // DROP SEQUENCE <sequence>
-	dropTable         // DROP TABLE <table>
-	dropEnumValue     // ALTER TYPE <type> DROP VALUE <value>
-	dropView          // DROP VIEW <view>
-	dropSchema        // DROP SCHEMA <schema>
-
-	primaryRegion //  ALTER DATABASE <db> PRIMARY REGION <region>
-
-	renameColumn   // ALTER TABLE <table> RENAME [COLUMN] <column> TO <column>
-	renameIndex    // ALTER TABLE <table> RENAME CONSTRAINT <constraint> TO <constraint>
+	renameIndex    // ALTER INDEX <table>@<index> RENAME TO <index>
 	renameSequence // ALTER SEQUENCE <sequence> RENAME TO <sequence>
 	renameTable    // ALTER TABLE <table> RENAME TO <table>
 	renameView     // ALTER VIEW <view> RENAME TO <view>
 
-	setColumnDefault // ALTER TABLE <table> ALTER [COLUMN] <column> SET DEFAULT <expr>
-	setColumnNotNull // ALTER TABLE <table> ALTER [COLUMN] <column> SET NOT NULL
-	setColumnType    // ALTER TABLE <table> ALTER [COLUMN] <column> [SET DATA] TYPE <type>
+	// The below list was generated by
+	// https://gist.github.com/chrisseto/cd5f94c7e70cbbccd9df05788e4b1cb8 and
+	// then hand curated.
+	// To aid in book keeping:
+	// Implemented commands are sorted alphabetically and then split into groupings.
+	// Unimplemented commands are sorted alphabetically.
+	// Disabled commands are handled by modifying opWeights.
+
```

Comment:
nit: Do you think it will helpful to add a line of comment here like
"Note: DDLs with non-zero weight in opWeights are generated and executed with legacy schema changer. Such DDLs can be occasionally executed with declarative schema changer if it's supported in `OpDeclarativeVersions`."

Reply:
Done!